### PR TITLE
added HandleCraftItem call to ShiftClickedResult to make sure achievements are awarded

### PR DESCRIPTION
**Pull request by 12xx12**
I was doing some changes at the achievements system as I noticed that crafting based achievements are not awarded if you shift click the crafting process

fixed it

### CODE REVIEW

**Comment by 12xx12** [comment]
Note: I didn't test this - time to sleep for today

**Comment by 12xx12** [comment]
Jup, works now

File: src/UI/SlotArea.cpp [hunk 1]
```diff
@@ -692,6 +692,9 @@ void cSlotAreaCrafting::ShiftClickedResult(cPlayer & a_Player)
 		// Broadcast the window, we sometimes move items to different locations than Vanilla, causing needless desyncs:
 		m_ParentWindow.BroadcastWholeWindow();
 
+		// added this to make sure that achievements are awarded
```
Comment (madmaxoft) [inline src/UI/SlotArea.cpp:695]: The comment could use some refactoring. "Added this" has no real meaning in a comment, we don't want to track some situational development, we want to know what the code is doing.
A simple `// Award achievements:` would suffice in this case.

Comment (12xx12) [reply src/UI/SlotArea.cpp]: @madmaxoft done

Comment (madmaxoft) [reply src/UI/SlotArea.cpp]: Sorry for being a nit-picker, the comment should start with a capital letter.

**Pull request merged**